sdks/python/apache_beam/ml/inference/base.py

```diff
@@ -174,6 +174,12 @@ def update_model_path(self, model_path: Optional[str] = None):
     """Update the model paths produced by side inputs."""
     pass
 
+  def validate_constructor_args(self):
```

Comment:
Sorry for not seeing this earlier, but do we actually get anything out of this being part of the base class? We're never going to call it from the base and its implementation will always be subclass dependent

Reply:
At first I kept this in the Pytorch model handler class but there was a suggestion at https://github.com/apache/beam/pull/25321#discussion_r1097855765. I might have understood wrong but what i thought is that, 

if we want to validate other handlers constructor args, lets share this method in the base ModelHandler and other subclasses can implement if they want

Reply:
Ah, I just meant to share it between PyTorch model handlers. Sorry, that was unclear. I think a single shared _validate_constructor_args function is enough here. Thanks for bearing with me here

Reply:
Sgtm. I will make changes